lib/ValidationError.ts

```diff
@@ -18,16 +27,18 @@ export class ValidationError extends Error {
   }
 }
 
-function fromZodIssue(
-  issue: zod.ZodIssue,
+function getMessageFromZodIssue(
```

Comment:
dev: previously `fromZodIssue` was used internally to format a `ZodIssue` as user-readable string. I've changed the name for this function to make more sense and release the former name for the new functionality currently implemented.